lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -590,7 +590,8 @@ def create_transaction_parameters(money, credit_card_or_vault_id, options)
               :number => credit_card_or_vault_id.number,
               :cvv => credit_card_or_vault_id.verification_value,
               :expiration_month => credit_card_or_vault_id.month.to_s.rjust(2, "0"),
-              :expiration_year => credit_card_or_vault_id.year.to_s
+              :expiration_year => credit_card_or_vault_id.year.to_s,
+              :cardholder_name => "#{credit_card_or_vault_id.first_name} #{credit_card_or_vault_id.last_name}"
```

Comment:
Really only because this was the way it was already being constructed for Apple Pay and I was in context. You're right that either would work.